### PR TITLE
feat: add ping command

### PR DESCRIPTION
**Pull request by vcokltfre**
## Relevant Issues
No github issues, but the relevant conversation in pydis/dev-contrib is around here https://discord.com/channels/267624335836053506/635950537262759947/831533475760832603

## Description
A simple command that sends an embed showing the bot's websocket latency. It is simple as was discussed; it has no need to be complex, just provide basic information and more importantly show that the bot is online.

## Screenshots
image

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

File: bot/exts/evergreen/ping.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    """Ping the bot to see its latency and state."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="ping")
+    async def ping(self, ctx: commands.Context) -> None:
+        """Ping the bot to see its latency and state."""
+        embed = Embed(
+            title="Pong!",
```
Comment (ToxicKidz) [inline bot/exts/evergreen/ping.py:17]: Needs a 🏓

**Comment by vcokltfre** [comment]
image

**Review by ToxicKidz — APPROVED** [review]
:shipit: :shipit: :shipit:

File: bot/exts/evergreen/ping.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="ping")
+    async def ping(self, ctx: commands.Context) -> None:
+        """Ping the bot to see its latency and state."""
+        embed = Embed(
+            title=":ping_pong: Pong!",
+            colour=Colours.bright_green,
+            description=f"WS Latency: {round(self.bot.latency * 1000)}ms",
```
Comment (jb3) [inline bot/exts/evergreen/ping.py:19]: Maybe we use Discord terminology here for clarity.
```suggestion
            description=f"Gateway Latency: {round(self.bot.latency * 1000)}ms",
```

**Review by jb3 — CHANGES_REQUESTED** [review]

**Review by jb3 — APPROVED** [review]

**Review by Kronifer — APPROVED** [review]
Everything checks out! It's a ping command all right :shipit:

**Review by Xithrius — APPROVED** [review]
Tested, it works well.

**Pull request merged**